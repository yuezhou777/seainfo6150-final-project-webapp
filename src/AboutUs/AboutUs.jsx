import React from 'react';
import NavBar from "../NavBar/NavBar";
import AboutUsImg from "../images/AboutUsPage/aboutus.jpeg";

import styles from "./AboutUs.module.css";

const AboutUs = () => {
    return(
        <div className={styles.container}>
            <div>
                <NavBar />
            </div>

            <div className={styles.board}>
                <div className={styles.aboutUsText}>
                    <h2 className={styles.aboutUsTitle}>About Us</h2>
                    <div className={styles.aboutUsContent}>
                        <p>We are people helping cats, dogs and wild animals go home and thrive. 
                            We do this by sheltering and adopting homeless cats and dogs, 
                            and educating the community to inspire compassionate action for animals.
                        </p>
                    </div>
                </div>
                <div>
                    <img className={styles.aboutUsImg} src={AboutUsImg} alt="About us image: a person hugs a dog"/>
                </div>
            </div>
        </div>
    );

}

export default AboutUs;