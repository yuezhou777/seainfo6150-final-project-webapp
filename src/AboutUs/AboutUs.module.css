.container {
    display: flex;
    flex-flow: column;
    font-family: 'Verdana', sans-serif;
}

.board {
    display: grid;
    grid-template-columns: 1fr 3fr;
}

.aboutUsTitle {
    text-align: left;
    color: saddlebrown;
}

.aboutUsImg {
    width: 100%;
    float: right;
}

.aboutUsText {
    text-align: left;
    margin: 40px;
    line-height: 40px;
}

.aboutUsContent {
    text-align:justify;
}

/* resposive */
@media (min-width: 400px) {
    .board {
        display: flex;
        flex-flow: row wrap;
    }
}

@media (min-width: 700px) {
    .board {
        display: flex;
        flex-flow: row wrap;
    }
}

@media (min-width: 800px) {
    .board {
        display: grid;
        grid-template-columns: 1fr 3fr;
    }
}