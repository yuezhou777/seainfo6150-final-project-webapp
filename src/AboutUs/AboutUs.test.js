import React from "react";
import { render } from "@testing-library/react";
import AboutUs from "./AboutUs.jsx";
import { BrowserRouter } from 'react-router-dom';

it("renders AboutUs Correctly", () => {

    const { container } = render(
        <BrowserRouter>
            <AboutUs />
        </BrowserRouter>
    );

    expect(container).toMatchSnapshot();
});