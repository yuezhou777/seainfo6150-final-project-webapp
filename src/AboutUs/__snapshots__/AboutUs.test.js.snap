// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AboutUs Correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div>
      <div
        class="navContainer"
      >
        <header
          class="navContainer"
        >
          <nav>
            <a
              class="navIcon"
              href="/"
            >
              PurrPet
            </a>
          </nav>
          <nav>
            <ul
              class="navList"
            >
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/aboutus"
                >
                  About Us
                </a>
              </li>
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/allpets"
                >
                  All Pets
                </a>
              </li>
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/category"
                >
                  Categories
                </a>
              </li>
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/contactus"
                >
                  Contact Us
                </a>
              </li>
            </ul>
          </nav>
        </header>
      </div>
    </div>
    <div
      class="board"
    >
      <div
        class="aboutUsText"
      >
        <h2
          class="aboutUsTitle"
        >
          About Us
        </h2>
        <div
          class="aboutUsContent"
        >
          <p>
            We are people helping cats, dogs and wild animals go home and thrive. We do this by sheltering and adopting homeless cats and dogs, and educating the community to inspire compassionate action for animals.
          </p>
        </div>
      </div>
      <div>
        <img
          alt="About us image: a person hugs a dog"
          class="aboutUsImg"
          src="aboutus.jpeg"
        />
      </div>
    </div>
  </div>
</div>
`;
